docs/src/tutorials/getting-started-jupyter/README.md

```diff
@@ -0,0 +1,11 @@
+# My first Jupyter Project
```

Comment:
Do you think the 'My first notebook' conclusion should now point to these tutorials as possible next steps?